### DISCUSSION TITLE
Add Spadedrive to Winget Tool

### DISCUSSION THREAD

**Discussion by alvarosamudio** [spacedriveapp/spacedrive#1580] [Feature Request]
Hi guys I am trying to make your cool program installable from the windows [winget](https://learn.microsoft.com/en-US/windows/package-manager/winget/) command this is one to install and manage applications, I think this would make this program even more accessible to Windows users allowing more people to know about it, I apologize in advance for [making the request to include the program](https://github.com/microsoft/winget-pkgs/pull/122568), I just see a lot of potential, I love it, and I am a somewhat lazy user I like the automation of software installation.

**Comment by utkubakir** [comment]
Thanks for letting us know @alvarosamudio, we are completely okay with having it on Winget. How will they know when we update it? Is that a manual process?
